src/components/modals/baseModal.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div v-if="show" class="">
```

Comment:
In PR #89 you were using vue Teleport, looks like it's missing here.

Reply:
There was a problem with testing Teleport but it should be solved now.